pom.xml

```diff
@@ -72,6 +83,7 @@
     </repositories>
 
     <build>
+        <directory>${buildDirectory}</directory>
```

Comment:
What is the purpose of this?

Reply:
need to specify different build paths for staging/prod and fallback for having different application.properties